adapters/adoppler/adoppler_test.go

```diff
@@ -7,6 +7,6 @@ import (
 )
 
 func TestJsonSamples(t *testing.T) {
-	bidder := NewAdopplerBidder("http://adoppler.com")
+	bidder := NewAdopplerBidder("http://{{.AccountID}}.trustedmarketplace.com")
```

Comment:
I'm a little sad to see the switch to account specific domains. This will greatly decrease the performance of your adapter on the server as the connection pool will become starved sooner. Is this really necessary?

Reply:
I'm afraid so. There is more like a client/tenant ID. There are no many clients (only a few of them). I just reused account ID to  avoid new variable introducing.

Reply:
:(

Ah well. That's fine on the repurposing of `AccountID`.